rpcs3/Emu/Cell/Modules/libmixer.cpp

```diff
@@ -6,6 +6,13 @@
 #include "cellAudio.h"
 #include "libmixer.h"
 
+#if __GNUC__ >= 6
+#include <math.h>
+#define FABS(X) std::fabs(X)
```

Comment:
You're right I used their defaults.
GCC 5.3 doesn't work with the default -std=gnu11 but it does with -std=c++11 instead.
The GNU dialect must be different somehow there.

The same option fixes Clang and ICC, so that seems good.
From a quick test same with -std=c++14, but ICC 13.0.1 that breaks, I guess it doesn't support it. Since I don't believe many use ICC is it ok to go with 14?